start.sh

```diff
@@ -25,4 +25,6 @@ cd ${INPUT_DIRECTORY}
 
 remote_repo="https://${GITHUB_ACTOR}:${INPUT_GITHUB_TOKEN}@github.com/${REPOSITORY}.git"
 
+git config --global --add safe.directory ${INPUT_DIRECTORY}
```

Comment:
Why do you set up the parameter global and inside the `start.sh`? For my understanding, it's better to do that inside the git config section.

```
    - name: Commit files
      run: |
        git config --local --add safe.directory <input_directory>
        git config --local user.email "41898282+github-actions[bot]@users.noreply.github.com"
        git config --local user.name "github-actions[bot]"
        git commit -m "Add changes" -a
```

Reply:
```suggestion
git config --local --add safe.directory ${INPUT_DIRECTORY}
```